test/fixtures/input/app/app.js

```diff
@@ -1,14 +1,16 @@
+import classic from 'ember-classic-decorator';
 import Application from '@ember/application';
 import Resolver from './resolver';
 import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
-const App = Application.extend({
-  modulePrefix: config.modulePrefix,
-  podModulePrefix: config.podModulePrefix,
-  Resolver,
-});
+@classic
+class AppApplication extends Application {
```

Comment:
Do we really need the name change here, can we just keep it App

Reply:
I should not have committed the integration test results. Whoops

Reply:
fixed